### PR TITLE
Learning Powershell one-liners

### PR DESCRIPTION
**Pull request by sharma-shruti**

### CODE REVIEW

**Comment by abhi18av** [comment]
@sharma-shruti , asking reviews on draft PRs is generally avoided as it signifies a `work-in-progress` :)

File: articles/Powershell_guide.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+REMARKS
+    Get-Help cannot find the Help files for this cmdlet on this computer. It is displaying only partial help.
+        -- To download and install Help files for the module that includes this cmdlet, use Update-Help.
+        -- To view the Help topic for this cmdlet online, type: "Get-Help Start-Process -Online" or
+           go to https://go.microsoft.com/fwlink/?LinkID=2097141.
+
+
+PS /root>
+```
+3. **Get-Member** - PowerShell deals with objects. Get-Member serves to help you identify what type of objects you are dealing with when using PowerShell and also what properties and methods are available to that object.
```
Comment (abhi18av) [inline articles/Powershell_guide.md:51]: You could also add content about the difference between a `bash` and `powershell` as in the former is based on text and the latter on objects; perhaps explain this with examples

File: articles/Powershell_guide.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+***
+```powershell
+PS /root/projects/perl_one_liners> Get-Date
+
+Thursday, 02 April 2020 09:44:37
+
+```
+## How to calculate the total value of third field in all the records?
+```powershell
+PS /root/projects/perl_one_liners> Get-Content names.txt | %{ [int]$total+=$_.Split(',')[2]; } ; Write-Host "Total: $total"
```
Comment (abhi18av) [inline articles/Powershell_guide.md:126]: Good one! 💯

File: articles/Powershell_guide.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+2,Nigel Shan Shanford,30316,20090405 16:34:00,400,9.99,3996,New-Opportunity
+3,Selma Cooper,97455,20090405 16:31:00,1000,9.99,9990,Pre-Approach
+4,Allen James,95140,20090405 16:31:00,1000,9.99,9990,New-Opportunity
+PS /root/projects/perl_one_liners> ((Get-Content ./names.txt) -replace 'Selma','Sheldon')
+1,Tony Passaquale,7920,20090222 21:59:00,800,4.78,3824,Follow-up
+2,Nigel Shan Shanford,30316,20090405 16:34:00,400,9.99,3996,New-Opportunity
+3,Sheldon Cooper,97455,20090405 16:31:00,1000,9.99,9990,Pre-Approach
+4,Allen James,95140,20090405 16:31:00,1000,9.99,9990,New-Opportunity
+```
+2) **Now that we have the code to find and replace the string we're after, it's now time to modify the file itself. We can do that by using Set-Content.**
```
Comment (abhi18av) [inline articles/Powershell_guide.md:153]: Instead of using `**` for bolds you could use the `#` or `##` to mark the line as a headline

**Review by abhi18av — APPROVED** [review]

**Review by abhi18av — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1 +1,3 @@
 .DS_Store
+*.ps1
```
Comment (abhi18av) [inline .gitignore:2]: This would ignore all powershell files - is this exactly what you want?

**Review by abhi18av — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (sharma-shruti) [reply .gitignore:2]: Yes, for the time being.

Comment (abhi18av) [reply .gitignore:2]: Cool 👍

File: articles/Powershell_guide.md (hunk 3 above)
Comment (abhi18av) [reply articles/Powershell_guide.md:153]: @sharma-shruti , please incorporate this one in future PRs

**Review by abhi18av — APPROVED** [review]
Added a suggestion for future PRs, merging this one.

**Pull request merged**